src/java/org/apache/cassandra/db/filter/RowFilter.java

```diff
@@ -224,7 +224,7 @@ protected BaseRowIterator<?> applyToPartition(BaseRowIterator<?> partition)
 
                 // Short-circuit all partitions that won't match based on static and partition keys
                 for (Expression e : partitionLevelExpressions)
-                    if (!e.isSatisfiedBy(metadata, partition.partitionKey(), partition.staticRow()))
+                    if (!e.isSatisfiedBy(metadata, partition.partitionKey(), partition.staticRow(), nowInSec))
```

Comment:
this is https://issues.apache.org/jira/browse/CASSANDRA-20108, needed to make the tests stable